spec/models/user_spec.rb

```diff
@@ -11,17 +11,17 @@
     expect(@user.email).to match 'user@example.com'
   end
 
-  it "#role is :user by default" do
-    expect(@user.role).to eq :user
+  it "#role is 'user' by default" do
+    expect(@user.role).to eq "user"
   end
 
-  it "#role can be set to :vip" do
+  it "#role can be set to 'vip'" do
```

Comment:
ohhh. i have no idea how to solve for that, except by doing something like `@user.role.to_sym` or something...